### PR TITLE
This is the spanish/english and game class

### PR DESCRIPTION
**Pull request by kannabanana**

### CODE REVIEW

File: src/main/java/models/Suit.java [hunk 1]
```diff
@@ -6,6 +6,6 @@
  * Created by michaelhilton on 1/25/16.
  */
 public enum Suit  implements Serializable {
-    Hearts, Spades, Diamonds, Clubs
+    Hearts, Spades, Diamonds, Clubs, Sword, Cups, Coins
```
Comment (athai) [inline src/main/java/models/Suit.java]: Sword -> Swords

You call plural "Swords" in your code, but it's been declared as singular in the suit class.

File: src/main/java/models/spanishgame.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ */
+public class englishGame extends Game{
+
+
+	for (int i = 1; i < 11; ++i){
+		deck.add(new Card(i,Suit.Clubs));
+		deck.add(new Card(i,Suit.Cups));
+		deck.add(new Card(i,Suit.Swords));
+		deck.add(new Card(i,Suit.Clubs));
+	}
```
Comment (leel8on) [inline src/main/java/models/spanishgame.java:19]: I think you meant to have a build deck function

File: src/main/java/models/Game.java [hunk 3]
```diff
@@ -37,13 +43,14 @@ public void buildDeck() {
...
             for (int i = 1; i < 11; i++){
-                deck.add(new Card(i,SpanishSuit.Coins));
-                deck.add(new Card(i,SpanishSuit.Cups));
-                deck.add(new Card(i,SpanishSuit.Swords));
-                deck.add(new Card(i,SpanishSUit.Clubs));
+                deck.add(new Card(i.Suit.Coins));
+                deck.add(new Card(i.Suit.Cups));
+                deck.add(new Card(i.Suit.Swords));
+                deck.add(new Card(i.Suit.Clubs));
             }
```
Comment (leel8on) [inline src/main/java/models/Game.java]: You replaced the commas with periods. Need to be commas

**Comment by athai** [comment]
You forgot to push your tests! D:

File: src/main/java/models/spanishgame.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Random;
+
+/**
+ * Created by michaelhilton on 1/25/16.
+ */
+public class englishGame extends Game{
+
```
Comment (leel8on) [inline src/main/java/models/spanishgame.java:11]: should this be spanishGame ?

File: src/test/java/models/testspanishGame.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  public class testspanishGame {
+
+	@Test
+	public void  testSpanishGameCreation(){
+		Game s = new SpanishGame();
+		assertNotNull(s);
+	}
+
+	@Test
+		public void testSpanishDeckCount() {
```
Comment (athai) [inline src/test/java/models/testspanishGame.java:20]: Your indentation is not consistent

File: src/main/java/models/spanishgame.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+public void buildDeck() {
+ 
+	for (int i = 1; i < 11; ++i){
+		deck.add(new Card(i,Suit.Clubs));
+		deck.add(new Card(i,Suit.Cups));
+		deck.add(new Card(i,Suit.Swords));
+		deck.add(new Card(i,Suit.Clubs));
```
Comment (athai) [inline src/main/java/models/spanishgame.java]: You've listed 'Clubs' twice. You should change one of them to 'Coins'.

**Comment by leel8on** [comment]
Still need to finish tests, but I think it is more important to get the code merged into master at this point.

**Pull request merged**